nixos/modules/system/boot/loader/limine/limine-install.py

```diff
@@ -113,6 +113,15 @@ def get_kernel_uri(kernel_path: str) -> str:
     return get_copied_path_uri(kernel_path, "kernels")
 
 
+@dataclass
+class XenBootSpec:
+    """Represent the bootspec extension for Xen dom0 kernels"""
+
+    xenEfi: str
+    xenMultiboot: str
+    xenVersion: str
+    xenParams: List[str]
```

Comment:
```suggestion
    efiPath: str
    multibootPath: str
    params: List[str]
    version: str
```
You updated the keys in the Xen bootspec but forgot to update the Limine install script, please update the rest of the script accordingly as this makes it not generate the Xen entries.

Reply:
Thanks, I did miss these when merging the two PRs back together. I've pushed a commit to make them consistent again, and also changed double quotes to single quotes in the changes, with the exception of the docstrings, or where double quotes have been used as part of log messages with the intent of the double quotes being present in the log messages themselves.